src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -134,6 +134,9 @@ class DefaultStorageKeyAllocator
     static StorageKeyName GroupDataCounter() { return StorageKeyName::FromConst("g/gdc"); }
     static StorageKeyName GroupControlCounter() { return StorageKeyName::FromConst("g/gcc"); }
 
+    // ICD Check-In Counter
+    static StorageKeyName ICDCheckInCounter() { return StorageKeyName::FromConst("icd/cic"); }
```

Comment:
g/icd/cic ?

Reply:
g was for group like group/**G**roup**D**ata**C**ounter --->g/gdc

So the g is not needed here and makes the key smaller

Reply:
@jepenven-silabs No, the "g/" there is for "global" and all keys _must_ start with "g/" or "f/".  This is an invariant that is: (1) depended on by SDK consumers and (2) documented in this file.